### ISSUE TITLE
Add Dark Mode theme toggle

### ISSUE THREAD

**Issue by RashikaKarki** [MLH-Fellowship/pod-3.1.1-portfolio#1]
It will be great if we can add a toggle to switch between dark and light mode in the template. 

I am thinking of something along the line of this image, maybe around the top right corner. 

![image](https://user-images.githubusercontent.com/41114269/125605388-4a84ead7-44d1-4b75-8a9e-a1d7ca270586.png)

**Comment by rosekamallove** [comment]
Hey, @RashikaKarki I @rosekamallove and @KUHOO-S will be working on this issue.

**Comment by rishavnaskar** [comment]
@rosekamallove After taking a pull of the latest code, I found a couple of issues in dark mode.

- [ ] The dark mode is not getting applied to projects.md file which was working fine previously
- [ ] An error is getting logged saying *ERROR `/js/themeSwitcher.js' not found*. I noticed the previously used file was removed from the main repo and maybe that is causing the error.

**Comment by rosekamallove** [comment]
Hey, @rishavnaskar  thanks for pointing out the issues, the issue is maybe you have an older version of the branch, I actually changed the location of the scripts suggested by @hkaur008 and you maybe have an older commit, So I actually did fix it already and If it still persists I will fork it and fix all the issue regarding the dark theme toggle.

**Comment by rishavnaskar** [comment]
> Hey, @rishavnaskar thanks for pointing out the issues, the issue is maybe you have an older version of the branch, I actually changed the location of the scripts suggested by @hkaur008 and you maybe have an older commit, So I actually did fix it already and If it still persists I will fork it and fix all the issue regarding the dark theme toggle.

Yes you are correct. I noticed that the *.js* files have been moved to the *assets* folder. I have the latest commits on my fork, but somehow the dark mode isn't applying to the project details page. Please let me know if you are facing the same.

**Comment by rishavnaskar** [comment]
@rosekamallove @KUHOO-S guys I am still facing this issue. Just reminding.
